docs/plugins/openid-connect.md

```diff
@@ -0,0 +1,206 @@
+## Introduction
+
+The page covering [authentication](../../rest-api/authentication) explains how to authenticate with a basic username/password for a user created directly in Mauro. An alternative authentication method is to use an [OpenID Connect](https://openid.net/connect/faq/) identity service to authenticate users with an external provider/account system, and then authorize them to use Mauro.
+
+In order use OpenID Connect identity providers, the Mauro instance must:
+
+1. Have the [Mauro OpenID Connect Authentication plugin](../../installing/plugins) installed.
+2. Set up and configure one or more identity services that support the OpenID Connect protocol.
+3. Add the configuration details of each identity provider to Mauro.
+
+The full details of this set up can be viewed in the user guide [Using OpenID Connect](../../user-guides/introduction).
```

Comment:
does the relevant pieces of information exist at this page??

Reply:
The user guide for Open ID connect is pending, notes on what it should include have been forwarded to Gemma (and @jamesrwelch was CC'd in too).